doc/classes/String.xml

```diff
@@ -181,6 +181,25 @@
 			<argument index="1" name="placeholder" type="String" default="&quot;{_}&quot;" />
 			<description>
 				Formats the string by replacing all occurrences of [code]placeholder[/code] with [code]values[/code].
+				[b]Note:[/b] There must be one and only one [code]_[/code] in the [code]placeholder[/code].
+				[codeblocks]
+				[gdscript]
+				# Use the defalut placeholder, and a two-dimensional array.
```

Comment:
Typo on "defalut".

Reply:
Reminds me of the time when I was learning English, well, not yet graduated in languages(including the native language) studies.